### ISSUE TITLE
Explanation of DCO not quite correct

### ISSUE THREAD

**Issue by richardfontana** [deptofdefense/code.mil#95]
In https://github.com/deptofdefense/code.mil/blob/master/Proposal/CONTRIBUTING.md#2-your-contributions-developers-certificate-of-origin, the statement "The DCO is a legally binding statement that asserts that you are the creator of your contribution and that you are intentionally making your contribution available under the license associated with the project." is not exactly consistent with what the DCO says, because the DCO takes into account the possibility that the contribution was created by someone else - see (b) and (c) of the DCO.

**Comment by shawoods** [comment]
@richardfontana Thank you for sharing your thoughts with us. I will admit, I'm pretty excited to see you jumping on the threads.

I agree that the statement is oversimplified and misses (b) and (c).  We've been struggling with trying to keep the explanations simple and in plain English without resulting in inaccurate statements. We were hoping to have a simple explanation recapping the effect of the DCO without repeating the DCO.  You make a valid point, though; we probably went too far here.  I will take another crack at it.

**Comment by richardfontana** [comment]
@shawoods to be sure, probably most cases of contributions to most projects will fall under part (a) of the DCO. Just wanted to be sure you all were aware of the issue.

**Comment by shawoods** [comment]
I just updated the language to be more inclusive of (b) and (c), but still trying to keep it concise.  You can check out the latest CONTRIBUTING.md.  Here is the language below:

> The DCO is a legally binding statement asserting that you are the creator of your contribution, or that you otherwise have the authority to distribute the contribution, and that you are intentionally making the contribution available under the license associated with the project.

What do you think?  Better?

**Comment by richardfontana** [comment]
Yes, that seems good.
